src/AppInstallerRepositoryCore/RepositorySource.cpp

```diff
@@ -926,6 +937,11 @@ namespace AppInstaller::Repository
                 bool sourceUpdated = false;
                 for (auto& source : currentSources)
                 {
+                    if (source.get().Restricted == true)

```

Comment:
> if (source.get().Restricted == true) [](http://example.com/codeflow?start=19&length=37)

Can you add a comment here like:
// Restricted sources may not support the full set of functionality so they shouldn't be included in the default aggregated source.